tests/sql/settings.yaml

```diff
@@ -0,0 +1,38 @@
+$schema: ../../json_schemas/test_story.schema.yaml
+
+description: Test updating SQL settings in OS cluster.
+
+chapters:
+  - synopsis: Update SQL settings.
+    path: /_plugins/_query/settings
+    method: PUT
+    request_body:
+      payload:
+        transient: 
+          plugins:
+            sql:
+              enabled: 'true'
```

Comment:
This should be not quoted since it's a boolean?

Reply:
Yeap! Sure, i fixed it